css/mainstyles.css

```
@@ -0,0 +1,412 @@
+  line-height: 160%;
+  color: #CDCDCD;
+  max-width: 460px;
+}
+
+.button {
+  background: #F1CDB3;
+  border-radius: 100px;
+  padding: 15px 45px;
+  border: none;
+  font-family: 'Georgia';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 130%;
+}
+
+.about {
+  font-family: 'Georgia';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 35px;
+  line-height: 130%;
+  color: #545454;
+  display: flex;
+  align-items: center;
+  letter-spacing: 0.06em;
+}
+
+.info p {
+  font-family: 'Arial';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 160%;
+  color: #4C4C4C;
+}
+
+ul,
+ol {
+  margin: 0;
+  padding: 0;
+}
+
+main {
+  display: flex;
+  align-items: center;
+}
+
+.ban-content {
+  max-width: 460px;
+}
+
+main img {
+  margin: 80px 0 0 42px;
+}
+
+.ban-content button {
+  margin-top: 42px;
+}
+
+.info {
+  display: flex;
+  flex-wrap: wrap;
+  padding-top: 80px;
+  padding-bottom: 100px;
+  justify-content: center;
+}
+
+.info-content {
+  margin-left: 120px;
+}
+
+.info p {
+  font-family: 'Arial';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 160%;
+  max-width: 430px;
+  margin-bottom: 25px
+}
+
+.catalog {
+  background-color: #F6F6F6;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.catalog .pets {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 90px 0 90px;
+  padding: 0 55px;
+  width: 100%;
+}
+
+.block-title {
+  font-family: 'Georgia';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 35px;
+  line-height: 130%;
+  color: #545454;
+  max-width: 400px;
+}
+
+.block-title.white {
+  color: #FFFFFF;
+}
+
+.petcard a {
+  font-family: 'Georgia';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 130%;
+  text-decoration: none;
+  color: #292929;
+  border: 3px solid #F1CDB3;
+  border-radius: 100px;
+  display: inline-block;
+  padding: 12px 47px;
+  margin: 30px;
+}
+
+.petcard p {
+  font-family: 'Georgia';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 23px;
+  margin: 30px 0;
+}
+
+.petcard {
+  background-color: #FAFAFA;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.itemsfrow,
+.itemssrow {
+  display: flex;
+  justify-content: space-around;
+  padding: 0 80px;
+}
+
+.itemsfrow {
+  gap: 120px;
+}
+
+.itemssrow {
+  padding: 50px 140px 100px;
+}
+
+.itemsfrow p,
+.itemssrow p {
+  font-family: 'Georgia';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 115%;
+}
+
+.itemsfrow .card,
+.itemssrow .card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.itemsfrow img,
+.itemssrow img {
+  max-width: 60px;
+}
+
+.help {
+  max-width: 1280;
+  max-height: 611px;
+}
+
+.donation {
+  display: flex;
+  background-color: #F6F6F6;
+  max-width: 1280px;
+  padding-top: 80px;
+  padding-bottom: 100px;
+  gap: 30px;
+  justify-content: center;
+}
+
+.donation .inform {
```

```suggestion
.inform {
```

Старайся избегать излишней вложенности селекторов - будет неудобно работать дальше, в том числе с адаптивом (переписать такой стиль можно только не менее сильным селектором - много букв, сложно читать)

В идеале нужно придумать класс, который будет использоваться только для этого типа элементов
А вложенность писать только когда в одном из блоков типовой элемент имеет какие-то особенности